internal/controller/pipelinerollout_controller.go

```diff
@@ -468,7 +468,7 @@ func (r *PipelineRolloutReconciler) processExistingPipeline(ctx context.Context,
 	// if not, should we set one?
 	if !inProgressStrategySet {
 		// inProgressStrategy is used for PPND and Progressive strategies (i.e. any strategies which require multiple reconciliations to perform)
-		if userPreferredStrategy == config.PPNDStrategyID && inProgressStrategy != apiv1.UpgradeStrategyPPND {
+		if userPreferredStrategy == config.PPNDStrategyID {
```

Comment:
realized these parts of the "if statement" were no longer needed now that this is enclosed within a check that `inProgressStrategy` isn't set at all